docs/UserGuide.md

```diff
@@ -45,30 +45,27 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
-
-* Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
-
-* Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[t/TAG]…​` can be used as ` ` (i.e. 0 times), `t/friend`, `t/friend t/family` etc.
+* Add personnel
+  * Add elderly
+  * Add volunteer
+* Delete personnel
+  * Delete elderly
+  * Delete volunteer
+* Add pairing of elderly and volunteer
+* Delete pairing of elderly and volunteer
+* View pairing list of elderly and volunteer
+* View personnel list
+  * View volunteer list
+  * View elderly list
 
-* Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
-
-* If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
-
-* Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
-  e.g. if the command specifies `help 123`, it will be interpreted as `help`.
```

Comment:
is there a reason why you removed this? seems quite useful

Reply:
I have added it back. Up to you to modify it since you are doing command format. I removed it initially assuming you wanted to start from scratch.
ps. Take our command format version from our user guide draft